### PR TITLE
Mobile v2.4 changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Review by cpoile — APPROVED** [review]

**Review by streamer45 — APPROVED** [review]

File: source/deploy/mobile-app-changelog.md [hunk 1]
```diff
@@ -2,12 +2,48 @@
...
+ - Added support for self-hosted Sentry URL parameters in build scripts.
+ - Added a ``files_count`` column to the ``ChannelInfo`` table and database migration.
+
+### Bug Fixes
+ - Fixed an issue where the app did not now show an error when adding servers without a diagnostic ID.
+ - Fixed an issue where users were unable to select several files when attaching files to a message.
+ - Fixed an issue where some channels would appear as if no messages were there.
+ - Fixed an issue with using the camera to capture photos/videos on Android versions 9 and below.
+ - Fixed a crash on iOS when attempting to open a Thread by tapping on the notification.
+ - Fixed an issue where channels would appear as empty.
```
Comment (larkox) [inline source/deploy/mobile-app-changelog.md:40]: This is the same issue as the one on line 37 "Fixed an issue where some channels would appear as if no messages were there.", but needed two different commits. We can delete any of the two lines. 0/5 on which one is more informative.

File: source/deploy/mobile-app-changelog.md [hunk 2]
```diff
@@ -2,12 +2,48 @@
...
+ - Added the "Add members" screen.
+ - Increased the character limit in the nickname field in the user interface from 22 to 64 characters.
+ - Added localization support for iOS usage description strings in permission dialogs.
+ - Users can now follow or unfollow a thread and undo the action if needed.
+ - Added a channel files screen.
+ - Added support for ``ExperimentalSettings.DelayChannelAutocomplete``.
+ - Calls: Call threads are now auto-followed when joining from mobile.
+ - Calls, Android: Fixed an issue with bluetooth; added audio device selection menu.
+ - Added support for self-hosted Sentry URL parameters in build scripts.
+ - Added a ``files_count`` column to the ``ChannelInfo`` table and database migration.
```
Comment (larkox) [inline source/deploy/mobile-app-changelog.md:32]: This one is more or less part of the effort on line 27: `Added a channel files screen.`. 0/5 on whether we should delete this line and just keep the one on line 27.

**Review by larkox — APPROVED** [review]
LGTM, just two potential changes, but nothing blocking.

File: source/deploy/mobile-app-changelog.md (hunk 1 above)
Comment (amyblais) [reply source/deploy/mobile-app-changelog.md]: Thanks, I removed the duplicate one.

File: source/deploy/mobile-app-changelog.md (hunk 2 above)
Comment (amyblais) [reply source/deploy/mobile-app-changelog.md]: Thanks, I removed line 32.

File: source/deploy/mobile-app-changelog.md [hunk 3]
```diff
@@ -2,12 +2,47 @@
...
+
+### Compatibility
+ - **Upgrade to server version v7.8.0 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/upgrade/extended-support-release.html) (ESR) v7.1.0 has ended and upgrading to server ESR v7.8.0 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
+ - iPhone 5s devices and later with iOS 12.1+ is required.
+
+### Improvements
+ - Added mobile support for message acknowledgements and persistent notifications.
+ - Added the "Add members" screen.
+ - Increased the character limit in the nickname field in the user interface from 22 to 64 characters.
```
Comment (cwarnermm) [inline source/deploy/mobile-app-changelog.md:24]:
```suggestion
 - Increased the character limit for the user's nickname from 22 to 64 characters.
```

**Review by cwarnermm — COMMENTED** [review]
Feedback (and questions) provided inline

File: source/deploy/mobile-app-changelog.md [hunk 4]
```diff
@@ -2,12 +2,47 @@
...
+ - Added localization support for iOS usage description strings in permission dialogs.
+ - Users can now follow or unfollow a thread and undo the action if needed.
```
[the remark this answers is not in the document]
Comment (amyblais) [reply source/deploy/mobile-app-changelog.md:26]: @cwarnermm - https://github.com/mattermost/mattermost-mobile/pull/7267

File: source/deploy/mobile-app-changelog.md [hunk 5]
```diff
@@ -2,12 +2,47 @@
...
+ - Added a channel files screen.
+ - Added support for ``ExperimentalSettings.DelayChannelAutocomplete``.
```
[the remark this answers is not in the document]
Comment (amyblais) [reply source/deploy/mobile-app-changelog.md]: Yes, server PR: https://github.com/mattermost/mattermost-server/pull/22952 & docs PR: https://github.com/mattermost/docs/pull/6346.

[the remark this answers is not in the document]
Comment (cwarnermm) [reply source/deploy/mobile-app-changelog.md]: Thanks!

**Pull request merged**